uwblueprint/community-fridge-kw
# feat(ui): donation information card refresh

## Brief description. What is this change? 

### [Update Calendar Card Component](https://www.notion.so/uwblueprintexecs/Update-Cards-for-Public-Calendar-and-Scheduled-Donations-da6885ba7ca3475fa63dd36e0e1e9254)
We're updating the information shown across the public calendar, donor dashboard, and admin calendar to be more consistent with the new designs.

Please see the figma for [updated designs for cards](https://www.figma.com/file/tzxZqXQRLYzAZGsSfgdw8o/?node-id=3%3A8103).

**Public Calendar**
![image](https://user-images.githubusercontent.com/19617248/159148705-e1b3bb18-b4a3-4dc7-8072-aafd8ed88672.png)
-------
**Admin Calendar**
![image](https://user-images.githubusercontent.com/19617248/159148674-d7f81759-764a-40ee-8c7a-67c554fb14a7.png)

**Donor Dashboard**
![image](https://user-images.githubusercontent.com/19617248/159148284-398d0d7d-2be5-4675-b8d1-ee112bdddd42.png)



## Steps to test
1. Create some schedules from different donors
2. Verify that the three views look correct on desktop and mobile
